test/extended/single_node/topology.go

```diff
@@ -0,0 +1,42 @@
+package single_node
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	exutil "github.com/openshift/origin/test/extended/util"
+	e2eskipper "k8s.io/kubernetes/test/e2e/framework/skipper"
+	"strings"
+)
+
+var _ = Describe("[sig-arch] Cluster topology single node tests", func() {
+	oc := exutil.NewCLIWithoutNamespace("singlenode").Verbose()
+
+	It("Verify that topology API is respected", func() {
+		out, err := oc.Run("get").Args("infrastructure", "cluster",
```

Comment:
use a real client, not an oc exec.

Reply:
Not sure what that means, can you elaborate?